.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           yarn install --cwd=acceptance-tests
       - name: Verify package dependencies
         run: |
-          npx depcheck --ignores=semver
+          npx depcheck --ignores="semver,@types/jest"
```

Comment:
I'm debating adding `@types/*` to the ignore because there are valid reasons to include types without ever importing them